### PR TITLE
Update Banners.json

### PR DESCRIPTION
**Pull request by QwQIsHidden**
3.2 Banners

## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by Birdulon** [comment]
This diff is unusable. It deletes Beginner's Banner, Character Event Banner 1, Character Event Banner 2 and Weapon Event banner, and inserts new Character Event and Weapon banners at the start of the file with some deprecated fields and some fields which literally don't exist.
Please use the existing file as a starting point and make the minimum required changes (i.e. `prefabPath`, `titlePath`, `rateUpItems5` and `rateUpItems4` for Character Event Banners 1&2 and Weapon Event Banner)

**Comment by YinCQL** [comment]
你的第二次提交仍有一点问题 (有个地方代码重复了...) ，你应该在原文件的基础上对于每个卡池仅修改这四个  “prefabPath” “titlePath” “rateUpItems4” “rateUpItems5” . 其他的保持不变。

File: src/main/resources/defaults/data/Banners.json [hunk 1]
```diff
@@ -34,49 +34,33 @@
...
+		"prefabPath": "GachaShowPanel_A103",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A081_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1071],
+		"rateUpItems1": [1073],
+		"rateUpItems2": [1073]
```
Comment (Birdulon) [inline src/main/resources/defaults/data/Banners.json:44]: `rateUpItems1` and `rateUpItems2` were deprecated 6 months ago. Additionally, you've set the featured 4-star pool to be the banner 5-star?

File: src/main/resources/defaults/data/Banners.json [hunk 2]
```diff
@@ -34,49 +34,33 @@
...
-		"titlePath": "UI_GACHA_SHOW_PANEL_A036_TITLE",
-		"costItemId": 223,
-		"beginTime": 0,
-		"endTime": 1924992000,
-		"sortId": 9998,
-		"rateUpItems4": [1072, 1065, 1053],
-		"rateUpItems5": [1022],
-		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
+		"weights5":[[1,75],[73,150],[90,10000]],
```
Comment (Birdulon) [inline src/main/resources/defaults/data/Banners.json:46]: Why are you deleting `Character Event Banner 2` and changing the weights to match Standard?

File: src/main/resources/defaults/data/Banners.json [hunk 3]
```diff
@@ -34,49 +34,33 @@
...
-		"titlePath": "UI_GACHA_SHOW_PANEL_A021_TITLE",
+		"prefabPath": "GachaShowPanel_A105",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A099_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
 		"sortId": 9997,
-		"rateUpItems4":[12415, 11405, 13407, 14403, 15401],
-		"rateUpItems5": [13511, 15503],
+		"rateUpItems4": [14511]
```
Comment (Birdulon) [inline src/main/resources/defaults/data/Banners.json:59]: Why are you setting the 4-star pool to one of the two featured 5-stars?

File: src/main/resources/defaults/data/Banners.json [hunk 4]
```diff
@@ -34,49 +34,33 @@
...
+		"rateUpItems5": [14511,15509],
 		"fallbackItems5Pool1": [],
 		"weights4": [[1,600], [7,600], [8,6600], [10,12600]],
-		"weights5": [[1,100], [62,100], [73,7800], [80,10000]],
+		"weights5":[[1,10000],[73,10000],[90,10000]],
```
Comment (Birdulon) [inline src/main/resources/defaults/data/Banners.json:63]: Why are you changing the weights? Why hard pity at 73? Why an additional curve point at 90 when 10000 has already been placed?

**Review by Birdulon — CHANGES_REQUESTED** [review]
I'm sorry but I can't understand where you are coming from at this point and I might just update the file myself.

**Pull request closed without merging**